HACKING.rst

```diff
@@ -55,7 +63,58 @@ server without requiring physical hardware. To suppress actual node and headnode
 operations, set ``dry_run = True`` in the ``[devel]`` section. For suppressing
 actual network switch operations, use the ``mock`` switch driver.
 
-Next initialize the database with the required tables, with ``haas-admin db create``.
+The file should be placed at ``/etc/haas.cfg``; The ``haas.wsgi``
+script, described below, requires this. Awkwardly, the ``haas``
+command line tool expects the file to be present in its current
+working directory. This will be fixed in the next release, but for
+now, put the file in ``/etc`` and create a symlink to it in the
+HaaS user's home directory::
+
+  sudo ln -s /etc/haas.cfg /var/lib/haas/
+  
+Additionally, the ``/etc/haas.cfg`` should have its permissions set to read-only and ownership
+set to the ``haas_user``::
+
+  sudo chown haas_user:haas_user /etc/haas.cfg
+  sudo chmod 400 /etc/haas.cfg
+
+Setting up Database
+===================
+
+If using SQLite no additional setup is necessary. Should ensure that `uri` in `database` setion of 'haas.cfg' is set to `sqlite:///haas.db` and can skip to initializing the database.
+
+If using PostgreSQL, follow these steps to setup the database:
+
+log into postgres account::
+
+  sudo -i -u postgres
+
+create user for the haas database::
+
+  createuser -Pd haas_user
+
+ -d allows haas_user to create databases
+ -P prompts to create a password
+
+create a database owned by the haas_user::
+
+  createdb haas_db -O haas_user
+  
+setup haas.cfg to access Postgres database:
+
+the ``uri`` option in haas.cfg must be set to::
+
+   postgresql://<user>:<password>@<address>/<dbname>
+
+Where ``<user>`` is the name of the postgres user you created (here this is ``haas_user``), ``<password>`` is its password, ``<dbname>`` is the name of the database you created (here this is ``haas_db``), and ``<address>`` is the address which HaaS should use to connect to postgres (In a typical default postgres setup, the right value is ``localhost``).
+ 
```

Comment:
delete all the lines including 81 to 110 since the information will get replaced by document added by #592